faunadb-common/src/main/java/com/faunadb/common/http/JavaHttpClient.java

```diff
@@ -0,0 +1,44 @@
+package com.faunadb.common.http;
+
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+import java.time.Duration;
+import java.util.concurrent.*;
+
+public class JavaHttpClient implements AutoCloseable {
+
+  private HttpClient _client;
+
+  public JavaHttpClient(int connectionTimeout) {
+    // TODO: [DRV-169] allow users to override default executor
+    this._client = HttpClient.newBuilder()
+      .connectTimeout(Duration.ofMillis(connectionTimeout))
+      .build();
+  }
+
+  public void close() {
```

Comment:
Seems like this class is only necessary to close over the connection timeout. Is it possible to remove this class and take a stdlib client directly?

Reply:
Yes, I think we could use it directly from the `Connection` now that it doesn't contain almost no _custom_ code.

Reply:
https://github.com/fauna/faunadb-jvm/pull/270/commits/1dc6fddf0846769ae86085ad8627b2302727ea51